### DISCUSSION TITLE
安装主题时出现syntax error，请大神赐教TT

### DISCUSSION THREAD

**Discussion by fredsun02** [solstice23/argon-theme#399] [General]
本人才疏学浅，之前没有接触过php代码。在Wordpress 5.9版本下安装主题，访问站点后提示致命错误：Parse error: syntax error, unexpected '?' in /www/wwwroot/frednotes.xyz/wp-content/themes/argon/functions.php on line 960
将此段代码注释掉之后会在其他位置报错。
请求先生们赐教，再谢。
<img width="537" alt="Capture" src="https://user-images.githubusercontent.com/96511411/153378837-584349ce-62d2-4b3a-9284-2d547c72a111.PNG">

**Comment by fredsun02** [comment]
不确定是不是bug..我估计是水平问题所致，因此没好意思发issue

**Comment by solstice23** [comment]
需要 php 7.0+

**Reply by fredsun02** [reply]
感谢回复。已解决！
